src/List.js

```diff
@@ -77,7 +78,14 @@ export function List() {
     const timeToMinDate = new Date() - minDate + ONE_MINUTE;
     //re-render the page so the item unchecks when it should be unchecked
     setTimeout(() => setReRender({}), timeToMinDate);
-  }, [reRender, items]);
+  }, [reRender, items, ONE_MINUTE]);
+
+  const handleDelete = async (id) => {
+    if (window.confirm('Are you sure you want to delete this item?')) {
+      await deleteDoc(doc(db, 'shopping-list', id));
+      setItems(items.filter((item) => item.id !== id));
+    }
+  };
```

Comment:
Maybe, we would like to create a new component (DeleteForm)with that part of the code. I think the list has started to become a really big component now, and maybe can you move this functionality to a new component?

Reply:
added!